### PR TITLE
Update in online compiler

### PR DESCRIPTION
**Pull request by veerbhadrasolanki**
Add Compiler in Java and C section

## What does this PR do?
Add resources in compilers.

### Description
Update compiler in java and c.

### Why is this valuable (or not)?
Because this provide us neat and clean in environment to do code.

### How do we know it's really free?
I was using this from last 2 year of my college and I never paid.

### For book lists, is it a book? For course lists, is it a course? etc.
No, it is not a book.

## Checklist:
- [x ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ x] Search for duplicates.
- [ x] Include author(s) and platform where appropriate.
- [ x] Put lists in alphabetical order, correct spacing.
- [ x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-playgrounds.md [hunk 1]
```diff
@@ -133,10 +133,10 @@
 
 ### Java
 
+* [Jdoodle](https://www.jdoodle.com/online-java-compiler/)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:136]:
```suggestion
* [JDoodle - Online Java Compiler IDE](https://www.jdoodle.com/online-java-compiler/)
* [JDoodle - Online Java Compiler Advanced IDE](https://www.jdoodle.com/online-java-compiler-ide/)
* [Programiz - Online Java Compiler](https://www.programiz.com/java-programming/online-compiler/)
```

File: more/free-programming-playgrounds.md [hunk 2]
```diff
@@ -46,9 +46,9 @@
 
 ### <a name="c"></a>C
 
+* [Programiz - C Compiler](https://www.programiz.com/c-programming/online-compiler/)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:49]:
```suggestion
* [JDoodle - Online C Compiler IDE](https://www.jdoodle.com/c-online-compiler/)
* [Programiz - Online C Compiler](https://www.programiz.com/c-programming/online-compiler/)
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: more/free-programming-playgrounds.md [hunk 3]
```diff
@@ -46,9 +46,9 @@
...
 * [SoloLearn](https://code.sololearn.com/c)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:51]:
```suggestion


```

File: more/free-programming-playgrounds.md [hunk 4]
```diff
@@ -133,10 +133,10 @@
 
 ### Java
 
+* [Jdoodle](https://www.jdoodle.com/online-java-compiler/)
 * [repl.it](https://repl.it) (_including a separate Java/Swing_)
 * [SoloLearn](https://code.sololearn.com/java)
```
Comment (davorpa) [inline more/free-programming-playgrounds.md:139]:
```suggestion


```

**Review by davorpa — CHANGES_REQUESTED** [review]
Restore removed lines to fix linter faults

**Review by davorpa — CHANGES_REQUESTED** [review]
Swap Java lines 138-139 to solve remaining linter faults: alphabetize list have to be ensured

**Review by veerbhadrasolanki — COMMENTED** [review]
Please Review

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**